c10/core/DispatchKey.h

```diff
@@ -123,58 +130,6 @@ enum class DispatchKey : uint8_t {
   // If you add new backend keys after PrivateUse3, please also update it here.
   EndOfBackendKeys = PrivateUse3,
```

Comment:
Just thinking out loud- is the Meta key above this so that's is a "real" backend just a semantic change? Or does it actually change behavior. I think the answer is that it doesn't change behavior- since Meta was already lower priority than the other non-backend keys, and it doesn't redispatch.

Reply:
It shouldn't be a behavior change, because (I think) there aren't any interactions between Meta and the keys I moved it past. Lack of redispatch from Meta helps a bit.